### DISCUSSION TITLE
Running "php artisan" throws error "Base table or view not found"

### DISCUSSION THREAD

**Discussion by kkoomen** [laravel/framework#34508] [General]
- Laravel Version: 6.18.38
- PHP Version: 7.2.23
- Database Driver & Version: mysql  Ver 14.14 Distrib 5.6.40, for Linux (x86_64) using  EditLine wrapper

### Description:

This didn't happen locally, but only when deploying to my CentOS 7 server.

```
# php artisan migrate

In Connection.php line 669:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'edu_gzpey_com.cache' doesn't exist (SQL: select * from `ca
  che` where `key` = meedu_cachetelescope:dump-watcher limit 1)


In PDOConnection.php line 72:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'edu_gzpey_com.cache' doesn't exist


In PDOConnection.php line 67:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'edu_gzpey_com.cache' doesn't exist
```

### Steps To Reproduce:

- Run `php artisan`

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. 

You can post support questions on our discussions tab instead so I'm moving it over there. I won't be able to help you out personally but others who are watching this repository can help you out.

Alternatively, you can post this question on one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

Thanks.
